src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -461,10 +503,14 @@ class Manager : public DoutPrefixProvider {
 
         entries_persistency_tracker& notifs_persistency_tracker = topics_persistency_tracker[queue_name];
         spawn::spawn(yield, [this, &notifs_persistency_tracker, &queue_name, entry_idx, total_entries, &end_marker,
-                             &remove_entries, &has_error, &waiter, &entry, &needs_migration_vector](spawn::yield_context yield) {
+            &remove_entries, &has_error, &waiter, &entry, &needs_migration_vector,
+            push_endpoint = push_endpoint.get()](
+            spawn::yield_context yield) {
             const auto token = waiter.make_token();
             auto& persistency_tracker = notifs_persistency_tracker[entry.marker];
-            auto result = process_entry(this->get_cct()->_conf, persistency_tracker, entry, yield);
+          auto result =
+              process_entry(this->get_cct()->_conf, persistency_tracker,
+                            entry, push_endpoint, yield);
```

Comment:
how does this work?
`process_entry()` expect a bare pointer, not a unique_ptr. shouldn't that be:
```
 process_entry(this->get_cct()->_conf, persistency_tracker,
                            entry, push_endpoint.get(), yield);
```